### PR TITLE
Make breadcrumbs configurable

### PR DESCRIPTION
**Pull request by annafatani**

### CODE REVIEW

**Review by jamesbirtles — CHANGES_REQUESTED** [review]

File: src/lib/breadcrumbs/BreadcrumbItem.svelte [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+		<slot />
+		<span class="pl-4 text-gray-400">&rsaquo;</span>
+	</div>
+{:else}
+	<div
+		class="inline-flex items-center text-sm font-medium text-gray-500 dark:text-gray-400 dark:hover:text-white"
+	>
+		<slot />
+	</div>
+{/if}
```
[the remark this answers is not in the document]
Comment (annafatani) [reply src/lib/breadcrumbs/BreadcrumbItem.svelte]: solved with contained styling

File: src/lib/breadcrumbs/Breadcrumb.svelte [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	}
+
+	li:last-child::after {
+		content: '';
+	}
+
+	li:last-child {
+		color: dimgray;
+	}
+</style>
```
Comment (jamesbirtles) [inline src/lib/breadcrumbs/Breadcrumb.svelte:21]: This should all be doable in tailwind,. for reference: https://tailwindcss.com/docs/content

File: src/lib/breadcrumbs/Breadcrumbs.svelte [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+	import type { SvelteHTMLElements } from 'svelte/elements';
+	import { twMerge } from 'tailwind-merge';
+
+	type $$Props = SvelteHTMLElements['section'];
```
Comment (jamesbirtles) [inline src/lib/breadcrumbs/Breadcrumbs.svelte:5]: this needs updating

File: src/lib/breadcrumbs/Breadcrumbs.test.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+test('should not render an arrow after the last breadcrumb item', () => {
+	const { getByText } = render(Breadcrumbs);
+	const lastItem = getByText(LAST_ITEM);
+	const arrow = within(lastItem).queryByText('>');
+	if (!arrow) {
+		return;
+	} else {
+		expect(arrow).toBeNull();
+	}
```
Comment (jamesbirtles) [inline src/lib/breadcrumbs/Breadcrumbs.test.ts:35]: The `if` shouldn't be necessary as this is the point of the `expect`

```suggestion
	expect(arrow).toBeNull();
```

File: src/lib/index.ts [hunk 5]
```diff
@@ -9,7 +10,7 @@ export * from './sortable';
 export { default as BackgroundImage } from './BackgroundImage.svelte';
 export { default as Banner } from './banner/Banner.svelte';
 export { default as BottomNav } from './BottomNav.svelte';
-export { default as Breadcrumbs } from './Breadcrumbs.svelte';
+export { default as Breadcrumbs } from './breadcrumbs/Breadcrumbs.svelte';
```
Comment (jamesbirtles) [inline src/lib/index.ts:13]: given that `export *` is being done, this is redundant

**Review by jamesbirtles — CHANGES_REQUESTED** [review]

**Comment by jamesbirtles** [comment]
also can you update the title so that it will look nice in the changelog

File: src/lib/breadcrumbs/Breadcrumb.svelte (hunk 2 above)
Comment (annafatani) [reply src/lib/breadcrumbs/Breadcrumb.svelte:21]: Thank you

**Review by jamesbirtles — APPROVED** [review]
Looks good 👍

**Pull request merged**